source/plugins/ruby/in_kube_nodes.rb

```diff
@@ -157,6 +155,19 @@ def parse_and_emit_records(nodeInventory, batchTime = Time.utc.iso8601)
         insightsMetricsEventStream = MultiEventStream.new
         kubePerfEventStream = MultiEventStream.new
         @@istestvar = ENV["ISTEST"]
+        
+        begin
+          # first refresh the node cpu and memory capacity cache
+          cpu_capacity_json = KubernetesApiClient.parseNodeLimits(nodeInventory, "capacity", "cpu", "cpuCapacityNanoCores")
+          @NodeCache.cpu.set_capacity_all(cpu_capacity_json)
```

Comment:
so the cache for all the nodes and not just windows nodes, is that right? if yes, why do need for Linux nodes?